<launch>
  <node name="zlac_driver_control" pkg="zlac_driver" type="driver_node.py" output="screen" respawn="true">
    <rosparam command="load" file="$(find zlac_driver)/config/zlac8015d.yaml"/>
  </node>

  <!-- <node pkg="joy" type="joy_node" name="joy">
      <param name="autorepeat_rate" value="5"/>
      <param name="deadzone" value="0.2"/>
  </node>

  <node pkg="teleop_twist_joy" name="teleop_twist_joy" type="teleop_node"/> -->
  
</launch>